src/DotNetProjectFile.Analyzers/Xml/PackageReference.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Xml.Linq;
+
+namespace DotNetProjectFile.Xml;
+
+public sealed class PackageReference : Node
+{
+    public PackageReference(XElement element) : base(element)
+    {
+    }
+
+    public string? Include => GetAttribute();
+
+    public string? Version => GetAttribute();
```

Comment:
Is it intentional that `Include` and `Version` are the same?

Reply:
``` C#
  public string? GetAttribute([CallerMemberName] string? propertyName = null)
        => Element.Attribute(propertyName)?.Value;
 ```
 
So the return the value of different attributes.

Reply:
In that case I think the properties shouldn't be nullable.

Reply:
What do you mean? Null means: attribute or element does not exist.